docs/3.0.0-beta.x/plugins/users-permissions.md

```diff
@@ -48,6 +48,11 @@ To change the default role, go to the `Advanced settings` tab and update the `De
 
 A jwt token may be used for making permission-restricted API requests. To make an API request as a user, place the jwt token into an `Authorization` header of the GET request. A request without a token, will assume the `public` role permissions by default. Modify the permissions of each user's role in admin dashboard. Authentication failures return a 401 (unauthorized) error.
 
+### JWT Configuration
+
+You can fully configure the JWT Configuration options by modifying the file in `strapi-plugin-users-permissions/config/jwt.json`.
```

Comment:
This is not completely true because you are not modifying this file directly, this file is in the `node_modules` of your application so you can't update this file.
I suggest we review together this paragraph by commenting.

Something like:

You can configure option for the JWT generation by creating `exetensions/users-permissions/config/jwt.json` file.
We are using [jsonwebtoken](https://www.npmjs.com/package/jsonwebtoken) to generate the JWT.
Available options:
- expiresIn: expressed in seconds or a string describing a time span zeit/ms.
Eg: 60, "2 days", "10h", "7d". A numeric value is interpreted as a seconds count. If you use a string be sure you provide the time units (days, hours, etc), otherwise milliseconds unit is used by default ("120" is equal to "120ms").